addons/stock/models/product.py

```diff
@@ -500,6 +500,16 @@ def action_update_quantity_on_hand(self):
     def action_product_forecast_report(self):
         action = self.env.ref('stock.report_stock_quantity_action_product').read()[0]
         action['domain'] = [('product_id', '=', self.id)]
+        action['context'] = {'default_product_id': self.id}
+        return action
+
+    def action_view_orderpoints(self):
+        action = self.env.ref('stock.product_open_orderpoint').read()[0]
+        if len(self) == 1:
```

Comment:
Do we have any case when this will called on Record sets ? (As It is a button on Form)

Reply:
Always better to support multi, even if it isn't made available as an action button.
But yes, It basically won't be called from a recordset of length > 1 unless another way of calling it is added.